datamade/just-spaces
# Premade intercept questions

## Overview

Connects #104 

This PR creates the following survey questions:

- [x] `age_intercept`: How old are you? 
**Integer**

- [x] `location_intercept`: What's the closest street intersection to where you live? 
**Holding off on this one until we get to  #81**

- [x] `gender_intercept`: What gender do you most identify with? 
**Dropdown: Male, Female, Non-binary, Choose not to answer**

- [x] `income`: What income group does your household fall under? 
**Dropdown: <20k, 20-40k, 40-60k, 60-80k, 80-100k, 100k+**

- [x] `education`: What is your highest formal degree of education? 
**Dropdown: Less than high school diploma, High school diploma, Associate's degree, Bachelor's degree, Graduate/professional degree**

- [x] `race`: Which race or ethnicity best describes you? 
**Dropdown: African-American/Black, Asian, Hispanic or Latino, Native American or Alaska Native, Native Hawaiian or Pacific Islander, Two or more races, White, Some other race**

- [x] `employment`: What is your current employment status?  
**Dropdown: Work full-time, Work part-time, Seeking work, Not seeking work, Student, Homemaker**

- [x] `household_tenure`: How many years have you lived at your current address?
 **Integer**

- [x] `own_or_rent`: Are you a homeowner or a renter? 
**Dropdown**: Renter, homeowner, other

- [x] `transportation`: How did you travel here? 
**Dropdown** ready to drop in these choices

Also to do:

## Testing Instructions

* If you don't already have it, create the database: `pg_restore -C -j4 --no-owner just-spaces.dump | psql`
* Migrate: `python manage.py migrate`
* Login as testuser (password in the DataMade LastPass under "Just Spaces Staging")
* Create a new Intercept survey
* Add each new element to it. Does the resulting element look like you'd expect?
